app/models/server/models/LivechatRooms.js

```diff
@@ -81,6 +82,21 @@ export class LivechatRooms extends Base {
 		return this.update(query, update);
 	}
 
+	updateBatchDataByToken(token, fields) {
+		const query = { 'v.token': token, 'open': true };
```

Comment:
changing the query to use an existing index:
```suggestion
		const query = { t: 'l', 'v.token': token, 'open': true };
```

Reply:
There's no index on t+v.token+open 🤔

Reply:
I'm suggesting one in the comment below 🙈